### ISSUE TITLE
Publish score rank to FIRST software

### ISSUE THREAD

**Issue by jpschewe** [jpschewe/fll-sw#570]
An initial integration would be to publish the score rank from FLL-SW to the message hub used by the FIRST software.

If we send something like this and the display system is up, we should see scores on the display screen.
```json
{
   "topic": "list:setArray",
   "data": {
       "header": "header on top",
       "data": [
           ["A1","B1","C1"],
           ["A2","B2","C2"],
           ["A3","B3","C3"]
       ]
   }
}
```
I expect that we could send such a list on each submitted score.

See documentation for the list module here: https://github.com/FirstLegoLeague/displaySystem#list

Generic documentation for mhub messages is at: https://github.com/FirstLegoLeague/displaySystem#advanced-usage

It looks like we send the following to make the list module show up on the display
```json
{
   "topic": "list:show"
}
```

Related to #473 

# Tasks

  * [x] add a global parameter to specify the address of the mhub server. Allow it to be cleared to not send any information to mhub
  * [x] add function to check if mhub parameter is set and to access the parameter
  * [x] Ui to set parameters
  * [x] when the default display is changed to the scoreboard, send the "list:show" message
  * [x] decide how to regularly update the mhub display, perhaps have a background timer run that continually cycles through the top 10 divisions and most recent scores
  * [x] handle return message from mhub
  * [x] `node` property needs to be a parameter. It needs to have a default vale of 'default'.
  * [x] support checking for acks to messages. Need to use the sequence numbers for this.
  * [x] Document how to use the functionality

**Comment by jpschewe** [comment]
I got an example working on my laptop. I cloned the displaySystem repository per the instructions at https://github.com/FirstLegoLeague/displaySystem#get-it-locally. 

I then installed mhub using the instructions at https://github.com/poelstra/mhub#basic-installation-and-usage. This installed and I started the server on the default port.

I then opened index.html from the displaySystem repository.

I have npm setup to install to ~/npm-global, so I used the following commands to show the list and change the text.
```shell-script
~/npm-global/bin/mhub-client -i -t "list:show"

~/npm-global/bin/mhub-client -i -t "list:setArray" -d "{\"header\": \"header on top\", \"data\": [ [\"A1\",\"B1\",\"C1\"],[\"A2\",\"B2\",\"C2\"],[\"A3\",\"B3\",\"C3\"]]}"
```
